frontend/packages/console-plugin-sdk/src/codegen/plugin-integration-tests.ts

```diff
@@ -0,0 +1,39 @@
+import * as _ from 'lodash';
+import {
+  PluginPackage,
+  filterActivePluginPackages,
+  PluginPackageFilter,
+  resolvePluginPackages,
+} from './plugin-resolver';
+
+const CONSOLE_PLUGIN_PREFIX = '@console/';
+const CONSOLE_INEGRATION_TESTS_PREFIX = '@console/integration-tests/';
+
+/**
+ * Return a map of integration tests.
+ */
+export const pluginIntegrationTests = (
```

Comment:
done.

p.s.
do we keep this convention everywhere in the code, is it a convention in our project ?

Reply:
It's just a convention, no need to modify existing code, but I'd use it for our new code.

`getSomething()` instead of just `something()` is more clear, IMO.